src/containers/org/sagas/deleteExistingOrganization.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * @flow
+ */
+
+import { call, put, takeEvery } from '@redux-saga/core/effects';
+import { OrganizationsApiActions, OrganizationsApiSagas } from 'lattice-sagas';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { AxiosUtils } from '../../../utils';
+import { DELETE_EXISTING_ORGANIZATION, deleteExistingOrganization } from '../actions';
+
+const { deleteOrganization } = OrganizationsApiActions;
+const { deleteOrganizationWorker } = OrganizationsApiSagas;
+
+const LOG = new Logger('OrgSagas');
+
+function* deleteExistingOrganizationWorker(action :SequenceAction) :Saga<void> {
+
+  try {
+    yield put(deleteExistingOrganization.request(action.id, action.value));
+
+    const org = action.value;
```

Comment:
Again, you don't need the org object, just call `deleteExistingOrganization(organizationId)`.